### DISCUSSION TITLE
Old version message

### DISCUSSION THREAD

**Discussion by sambo57u** [fjwillemsen/NativeOverleaf#24] [General]
Hi, I just installed the Linux x86_64 version on my computers. When I start the application I get a message saying this is a very old version and it is insecure to use etc. Is the downloaded version for x86_64 really that old? Is it possible to have the latest version? Thanks!

**Comment by fjwillemsen** [comment]
Hi, apologies for this warning. It's inserted by Nativefier (the framework we use to compile the app) to prevent outdated builds. Normally I always update Native Overleaf as soon as a new version of Nativefier is available, but their own software has not been updated since March, so there simply is no newer version available. Nevertheless, I have just released a [new version of Native Overleaf ](https://github.com/fjwillemsen/NativeOverleaf/releases/latest/) that will fix this for the time being.
